src/components/WwwFrame/TheHeader.vue

```diff
@@ -77,10 +77,7 @@
 							data-testid="header-log-in"
 							:event="showPopupLogin ? '' : 'click'"
 							@click.native="auth0Login"
```

Comment:
These lines should be removed:
```
:event="showPopupLogin ? '' : 'click'"
@click.native="auth0Login"
```
Be sure to also remove them from other router-link on line 312.

The showPopupLogin computed value and the auth0login method should be removed also, since those won't be used anymore.

Reply:
Done!